frontend/src/services/notebookService.ts

```diff
@@ -15,7 +15,7 @@ import {
 import { RecursivePartial } from '../typeHelpers';
 import { LIMIT_NOTEBOOK_IMAGE_GPU } from '../utilities/const';
 import { MOUNT_PATH } from '../pages/notebookController/const';
-import { usernameTranslate } from 'utilities/notebookControllerUtils';
+import { usernameTranslate } from '../utilities/notebookControllerUtils';
```

Comment:
🤔 I wonder if we should look to have more paths, not less... save us from `..` hell. Maybe not for your PR though. Food for thought.